code/modules/karma/karma.dm

```diff
@@ -117,6 +120,20 @@ var/list/karma_spenders = list()
 
 	spend_karma(pickedmob)
 
+/mob/verb/toggle_karma()
+	set name = "Toggle Karma Gains"
+	set desc = "Unlocked everything? Got too much karma? This button will let you disable karma gains!"
```

Comment:
"This button will allow you to stop other people giving you karma."
Reason: it makes it clear that anyone attempting to do so will fail, rather than letting them try but your karma score simply not going up.